content/200-orm/800-more/600-help-and-troubleshooting/100-help-articles/400-nextjs-prisma-client-dev-practices.mdx

```diff
@@ -1,12 +1,12 @@
 ---
-title: 'Best practices for using Prisma Client with Next.js'
-metaTitle: 'Best practices for using Prisma Client with Next.js'
-metaDescription: 'Best practices for using Prisma Client with Next.js'
+title: 'Best practice for instantiating PrismaClient with Next.js'
```

Comment:
missing space in Prisma Client `PrismaClient`

Reply:
I think that might be intentional, as it refers to `new PrismaClient()`?
Maybe adding the `()` would be helpful then?

Reply:
I think it's intentional as well, but I don't agree with it. I would prefer to have `instantiating Prisma Client` but if we feel the existing is better then `PrismaClient()` would probably be better.